//package core.io.protocol;

message BattleEnemyUnit
{
	optional string 	  id        		= 1;//式神ID
	optional int32		  role				= 2;//1攻击方 2防守方
	optional bool 	  	  cri           	= 7;//是否暴击
	optional bool 	  	  hit           	= 8;//是否命中
	optional int32 	  	  damage        	= 9;//伤害值
	optional int32		  shield			= 10;//护盾减免伤害数量
	optional int32		  enemyeffecttype	= 11;//对敌效果类型
	optional int32		  enemyeffectvalue	= 12;//对敌效果数值
	optional int32		  reservepos		= 13;//替补索引 -1 0 1
}


